packages/test/src/playwright.ts

```diff
@@ -7,9 +7,40 @@ export interface PlaywrightTestContext<chain extends Chain = Chain> {
   client: AnvilTestClient<chain>;
 }
 
+const attachLogToPlaywright = (testInfo: TestInfo) => {
+  const anvilLogs: string[] = [];
+
+  const attachLog = (message: string) => {
+    anvilLogs.push(message);
+  };
+
+  const onClose = (message: string) => {
+    attachLog(message);
+
+    testInfo.attach("Anvil Logs", {
+      body: anvilLogs.join("\n"),
+      contentType: "text/plain",
+    });
+  };
+
+  return {
+    onMessage: attachLog,
+    onError: attachLog,
+    onClose: onClose,
+  };
+};
```

Comment:
To me this should be end user code because we're preventing devs from passing their own callbacks to `createViemTest`

I would simply expect the same optional argument `callbacks?: AnvilProcessCallbacks`

And for its vitest counterpart too! (also named `createViemTest`)

Reply:
Don't you think that making it native to your playwright wrapper is a good use case? 
But happy to move it wherever you want or even outside of the repo.

Reply:
We can export the resulting object (the 3 callbacks) as a constant `attachLogs` from this file if you want! So you just have to pass it as the third optional parameter, in place of callbacks